### PR TITLE
Fix path of git-filter-repo.(1|html)

### PR DESCRIPTION
**Pull request by mattiasdrp**
Since they are not at the root of the repo but in Documentation/(man1 | html) (the Makefile puts them here and the last releast tarball has them in it too)

### CODE REVIEW

**Review by newren — CHANGES_REQUESTED** [review]
Hi,

Sorry for taking so long to respond.

The changes make sense.  The commit message needs a little work -- lines should be wrapped at 72 characters, and I need a Signed-off-by line and a real email address to include the changes.  Could you update the commit to fix these?

**Comment by mattiasdrp** [comment]
Hi,
No worries, we all have a lot of things to do :-)
I fixed my commit to include the changes
Thanks for this nice package

**Comment by newren** [comment]
Hi,

Thanks for understanding, and I'm glad you've enjoyed the package.

Thanks for fixing the wrapping and adding a Signed-off-by.  It appears you've used a username rather than a real name, though.  Is that right?  As per https://github.com/git/git/blob/4fd6c5e44459e6444c2cd93383660134c95aabd1/Documentation/SubmittingPatches#L284-L286, I unfortunately need a real name for signoff.  (And the author and committer should match the Signed-off-by as well.)  Can you fix that?

**Comment by mattiasdrp** [comment]
I'm not very fond of letting my real name everywhere on the Internet. I don't mind you taking the ownership of this PR and merging it with your name though so ;-)

**Pull request closed without merging**
